src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,234 @@
+package duke;
+
+import duke.exceptions.IncorrectFormatException;
+import duke.exceptions.UnrecognizedCommandException;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Todo;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    static TaskManager taskManager = new TaskManager();
+    static Scanner scanner = new Scanner(System.in);
+    static FileDataManager fileDataManager = new FileDataManager();
+
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
```

Comment:
Consider refactoring this logo into a `final` variable. Avoid the use of magic literals.